modules/ingest-geoip/src/internalClusterTest/java/org/elasticsearch/ingest/geoip/GeoIpDownloaderIT.java

```diff
@@ -254,7 +263,7 @@ public void testUseGeoIpProcessorWithDownloadedDBs() throws Exception {
             for (Path geoipTmpDir : geoipTmpDirs) {
                 try (Stream<Path> list = Files.list(geoipTmpDir)) {
                     List<String> files = list.map(Path::toString).collect(Collectors.toList());
-                    assertThat(files, containsInAnyOrder(endsWith("GeoLite2-City.mmdb"), endsWith("GeoLite2-Country.mmdb"),
+                    assertThat(files, hasItems(endsWith("GeoLite2-City.mmdb"), endsWith("GeoLite2-Country.mmdb"),
```

Comment:
maybe we should ensure that the text files exist as well?

Reply:
This is good idea, I'll change that

Reply:
Done